pkg/controller/servicebindingrequest/retriever_test.go

```diff
@@ -16,338 +19,52 @@ func TestRetriever(t *testing.T) {
 	ns := "testing"
 	backingServiceNs := "backing-servicec-ns"
 	crName := "db-testing"
-	testEnvVarPrefix := "TEST_PREFIX"
-	emptyEnvVarPrefix := ""
+	// testEnvVarPrefix := "TEST_PREFIX"
 
 	f := mocks.NewFake(t, ns)
 	f.AddMockedUnstructuredCSV("csv")
 	f.AddNamespacedMockedSecret("db-credentials", backingServiceNs)
 
-	crdDescription := mocks.CRDDescriptionMock()
 	cr, err := mocks.UnstructuredDatabaseCRMock(backingServiceNs, crName)
 	require.NoError(t, err)
 
 	crInSameNamespace, err := mocks.UnstructuredDatabaseCRMock(ns, crName)
 	require.NoError(t, err)
 
-	plan := &Plan{
-		Ns:   ns,
-		Name: "retriever",
-		RelatedResources: []*RelatedResource{
-			{
-				CRDDescription: &crdDescription,
-				CR:             cr,
-			},
-			{
-				CRDDescription: &crdDescription,
-				CR:             crInSameNamespace,
-			},
+	serviceCtxs := ServiceContextList{
+		{
+			Object: cr,
 		},
-	}
-
-	fakeDynClient := f.FakeDynClient()
-
-	retriever = NewRetriever(fakeDynClient, plan, "SERVICE_BINDING")
-	require.NotNil(t, retriever)
-
-	t.Run("getCRKey", func(t *testing.T) {
-		imageName, _, err := retriever.getCRKey(cr, "spec", "imageName")
-		require.NoError(t, err)
-		require.Equal(t, "postgres", imageName)
-	})
-
-	t.Run("read", func(t *testing.T) {
```

Comment:
the number of tests got remove here - with what tests are they replaced?

Reply:
Functionality has been removed from the type, thus tests have been removed.